run_bootstrap.sh

```diff
@@ -5,7 +5,7 @@ IP=$(ip a | grep "inet " | grep -Fv 127.0.0.1 | sed 's/.*inet \([^/]*\).*/\1/')
 echo "I am a bootstrap node"
 
 exec /usr/bin/wakunode\
-      --relay=true\
+      --relay=false\
       --rest=true\
       --rest-admin=true\
       --rest-private=true\
```

Comment:
can we remove --keep-alive=true\ as well?

Reply:
I tested without it and everything still seems to work. Removed it in https://github.com/waku-org/waku-simulator/pull/80/commits/b144e1bc16bea2f4fd5e2c728fed3ee19cab2793